src/lib/carpet.js

```diff
@@ -0,0 +1,11 @@
+/**
+* Copyright 2012-2017, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+module.exports = require('../src/traces/carpet');
```

Comment:
Maybe it shouldn't be included by default, but see: https://github.com/plotly/plotly.js/blob/3f18627be8cb1be48a9c3450329e0cbed713ffd6/lib/index.js#L41